eng/Subsets.props

```diff
@@ -27,7 +27,7 @@
        flavor is used to decide when to build the hosts and installers. -->
   <PropertyGroup>
     <!-- Determine if the CoreCLR runtime can build/run for the specified target. -->
-    <_CoreCLRSupportedOS Condition="'$(TargetsMobile)' != 'true' and '$(TargetsLinuxBionic)' != 'true'">true</_CoreCLRSupportedOS>
+    <_CoreCLRSupportedOS Condition="'$(TargetsAndroid)' == 'true' or ('$(TargetsMobile)' != 'true' and '$(TargetsLinuxBionic)' != 'true')">true</_CoreCLRSupportedOS>
```

Comment:
```suggestion
    <_CoreCLRSupportedOS Condition="'$(TargetsMobile)' != 'true' and '$(TargetsLinuxBionic)' != 'true'">true</_CoreCLRSupportedOS>
```

Reply:
This is going to be needed to trigger the runtime pack generation. I can see reverting this and following up.